src/cpr/create_or_update_pull_request.py

```diff
@@ -56,25 +56,37 @@ def create_or_update_pull_request(
     team_reviewers,
     project_name,
     project_column_name,
+    request_to_parent,
 ):
+    if request_to_parent is None:
+        request_to_parent = False
+    else:
+        request_to_parent = request_to_parent.lower() in ['true', '1', 't', 'y', 'yes', 'on']
+
+    github_repo = head_repo = Github(github_token).get_repo(github_repository)
+    if request_to_parent:
+        github_repo = github_repo.parent
+        if github_repo is None:
+            raise ValueError("The repository is not a fork. The parameter request-to-parent should be set to false.")
```

Comment:
```suggestion
            raise ValueError("The checked out repository is not a fork. Input 'request-to-parent' should be set to false.")
```

Reply:
Sorry, I missed this one. I made the suggestion on the `dist` version and forgot the actual source.

Reply:
good catch!
duplicate code make things hard. 
Is there a way to avoid it? symlink? Working on src only and import in dist?..

Reply:
It's built by `npm run package` so in general it's best to use that to generate the dist version. But there is not really any good way to avoid it showing up in the diff. It's not a big problem though. 😄